licenses/bx_python.txt

```diff
@@ -0,0 +1,23 @@
+The file skbio/metadata/tests/test_intersection.py, skbio/metadata/_intersection.c,
```

Comment:
Remove this paragraph, I don't think the original bx-python license should be modified (we point to it in the relevant code).